recipes/newlib/recipe.sh

```diff
@@ -24,7 +24,7 @@ function recipe_build {
         automake-1.11 --cygnus Makefile
     popd
 
-    CC= ./configure --target="${HOST}" --prefix=/
+    CC= ./configure --target="${HOST}" --prefix=/ --enable-newlib-iconv
```

Comment:
I suppose this should also be changed in https://github.com/redox-os/libc/blob/master/setup.sh